### PR TITLE
Add v1.22.0 changelog

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,169 @@ This changelog goes through the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.com).
 
+## [1.22.0](https://github.com/go-gitea/gitea/releases/tag/v1.22.0) - 2024-05-24
+
+Since there are too many pull requests, this changelog will ignore bugfixes,
+misc and etc changes.
```
Comment (yp05327) [inline CHANGELOG.md]: add a link to https://github.com/go-gitea/gitea/pulls?q=is%3Apr+milestone%3A1.22.0+is%3Amerged ?
Then user can check all PRs quickly if they want.

**Review by yp05327 — APPROVED** [review]
Thanks for the works from all contributors and maintainers.

File: CHANGELOG.md (hunk 1 above)
Comment (lunny) [reply CHANGELOG.md]: 80ec02a

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,171 @@ This changelog goes through the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.com).
 
+## [1.22.0](https://github.com/go-gitea/gitea/releases/tag/v1.22.0) - 2024-05-24
+
+This release includes [1478](https://github.com/go-gitea/gitea/pulls?q=is%3Apr+milestone%3A1.22.0+is%3Amerged) pull requests, which makes it the largest release in Gitea's history! It inclues a mount of new features and bug fixes.
```
Comment (metiftikci) [inline CHANGELOG.md]:
```suggestion
This release includes [1478](https://github.com/go-gitea/gitea/pulls?q=is%3Apr+milestone%3A1.22.0+is%3Amerged) pull requests, which makes it the largest release in Gitea's history! It includes a number of new features and bug fixes.
```

i am not native speaker. just a suggestion

**Review by appleboy — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (lunny) [reply CHANGELOG.md]: 43f2d8b

**Pull request merged**